packages/lib/constants.ts

```diff
@@ -53,3 +56,6 @@ export const IS_STRIPE_ENABLED = !!(
 );
 /** Self hosted shouldn't checkout when creating teams unless required */
 export const IS_TEAM_BILLING_ENABLED = IS_STRIPE_ENABLED && (!IS_SELF_HOSTED || HOSTED_CAL_FEATURES);
+export const CalComVersion = `v.${pkg.version}-${
+  process.env.NEXT_PUBLIC_WEBSITE_URL === "https://cal.com" ? "h" : "sh"
+}`;
```

Comment:
can we add here also if someone has a commercial license or not? we may need to move this from const into a proper component

`session.data?.hasValidLicense ? "l" : "n"` (l = license, n = no license)

Reply:
i.e.: v.2.4.2-sh-l

Reply:
the problem with session.data is that it's false when logged out because session is being destroyed when logging out

@zomars is there a way to know if someone has a EE license without checking session?

Reply:
i am now checking for process.env.CALCOM_LICENSE_KEY is empty which should be enough. people could enter an invalid key but why would they? doesnt benefit them since the EE components need a license